build-all-targets.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 #
-# Copyright (c) 2023 Contributors to the Eclipse Foundation
+# Copyright (c) 2023-2024 Contributors to the Eclipse Foundation
```

Comment:
According to https://www.eclipse.org/projects/handbook/#ip-copyright-headers adding a range of years no longer seem to be needed. They mention only "The {year} is the year of the initial creation"

Reply:
Fixed